src/Sylius/Behat/Page/Shop/Product/ShowPageInterface.php

```diff
@@ -83,4 +83,6 @@ public function visit(string $url): void;
     public function getVariantsNames(): array;
 
     public function getOptionValues(string $optionName): array;
+
+    public function getDetails(): string;
```

Comment:
The tab is called "details" but I believe we're checking a `description` indeed, so we should have `getDescription()` function 🖖